backend/workers/lib/vod_converter/split_labels_from_file.py

```diff
@@ -0,0 +1,64 @@
+import json
+import argparse
+import sys
+import os
+import collections
+
+
+def main(labels_to_split, output_dir, labels_per_file):
+    os.makedirs(output_dir, exist_ok=True)
+
+    with open(labels_to_split) as file:
+        data = json.load(file)
+
+    images = data['images']
+    chunk = int(len(images) / labels_per_file)
+    print("Splitting images")
+    image_chunks = [images[i * labels_per_file:(i + 1) * labels_per_file] for i in range(chunk)]
+    image_chunks.append(images[chunk * labels_per_file:])
+
+    print("Finding images id")
+
+    print("Splitting annotations")
```

Comment:
blank lines and prints

Reply:
Script is actually redundant so it was removed